recipe/meta.yaml

```diff
@@ -18,13 +18,13 @@ build:
 
 requirements:
   build:
-    - python >=3
-    - setuptools
+    - python >=3.6
     - pip
   run:
-    - python >=3
-    - azure-common >=1.1
-    - msrestazure >=0.4.11
+    - python >=3.6
+    - msrest >=0.6.21
+    - azure-common ~=1.1
+    - azure-mgmt-core >=1.3.0,<2.0.0
```

Comment:
https://github.com/Azure/azure-sdk-for-python/blob/azure-mgmt-storage_20.0.0/sdk/storage/azure-mgmt-storage/setup.py#L68-L71